src/CoreWCF.Primitives/src/CoreWCF/Security/SecuritySessionServerSettings.cs

```diff
@@ -851,7 +862,7 @@ private abstract class ServerSecuritySessionChannel : /*ChannelBase,*/ IServerSe
             // Double-checked locking pattern requires volatile for read/write synchronization
             private bool _areFaultCodesInitialized;
             //private readonly IServerReliableChannelBinder _channelBinder;
-            private readonly SecurityProtocol _securityProtocol;
+            protected readonly SecurityProtocol _securityProtocol;
 
```

Comment:
Expose via protected property instead of field

Reply:
done